### ISSUE TITLE
Feature request: Bronze and Steel items

### ISSUE THREAD

**Issue by Atlantispy** [MythrilBagels/Create-Alloyed#20]
I'd very much appreciate armour sets with the new material, given the significance of the bronze age in history, and steel being used in full plate armour later in history. (They'd also fit very nicely decoratively on armour stands)

Why your idea is unique (We don't want people to repeat each other): I did check there wasn't an existing suggestion of this, though I'm sure someone has thought of it before.
Why it fits the theme of Create and Create: Alloyed. Given the existence of steel tools, it seems to fit the extension of these materials.

Thanks

**Comment by FortressNebula** [comment]
Firstly, thank you for following the template!
Bronze armour isn't planned because Bronze as a metal was designed to be a decorative metal, not for armour or tools. I agree that this seems odd considering the Bronze Age and it's historical significance, but giving Bronze decorative *and* industrial capabilities would be a bit much.
However, Steel armour is a possibility. We're currently working on other features, but yes, Steel armour is a good idea and could come to the mod after we're done on the other steel stuff.

TL;DR: Bronze Armour is a no, Steel Armour is a maybe

**Comment by FortressNebula** [comment]
Thank you for the suggestion though, getting feedback is always nice :)

**Comment by Atlantispy** [comment]
Thanks! If I can ever convince you let me know, I'd love to have a museum with bronze armour talking about smelting practices and stuff :)
